nixos/modules/services/web-apps/gancio.nix

```
@@ -0,0 +1,273 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.gancio;
+  isLocalHostName =
+    cfg.hostName == "localhost"
+    || (lib.hasSuffix ".local" cfg.hostName)
+    || (lib.hasSuffix ".localhost" cfg.hostName);
+  json = pkgs.formats.json { };
+in
+{
+  options.services.gancio = {
+    enable = lib.mkEnableOption (
+      lib.mdDoc ''
+        A shared agenda for local communities.
```

This will be prefixed with `Wether to` and suffixed with `.`.

```suggestion
        Gancio, a shared agenda for local communities.
```